### ISSUE TITLE
Fetcher concurrency issue

### ISSUE THREAD

**Issue by popravich** [aio-libs/aiokafka#28]
There is a problem with `Fetcher`, I don't know if its bad design or simply "ported as is" issue, anyway:
`fetched_records` and `next_record` methods use `_wait_empty_future` to halt until results are read,
but this would only work if you had only one consumer task running. If you'd need to run several
asyncio tasks for consuming and processing messages you'll definetly end up with
`_wait_empty_future` being overriden with other one from other task and thus leave first task
waiting for future's result until loop is closed.

https://github.com/aio-libs/aiokafka/blob/master/aiokafka/fetcher.py#L604
Basically there is no check if `_wait_empty_future` already set (not None)

**Comment by fabregas** [comment]
please see #29
